themes/Afterglow.conf

```diff
@@ -18,4 +18,4 @@ color6                #7dd5cf
 color14               #7dd5cf
 color7                #d0d0d0
 color15               #f5f5f5
-selection_foreground #202020
+selection_foreground  #232323
```

Comment:
The original theme doesn't define a color for the foreground selection, in this case, I think that Sublime defaults to the foreground color for unselected text.